### PR TITLE
Feat archive delete

### PR DESCRIPTION
**Pull request by shubhamkumar648**
## add delete funtionality

- move to trash 
- add to trash
chrome-capture-2022-4-24

- delete from trash

### CODE REVIEW

File: src/component/notecard/Notecard.js [hunk 1]
```diff
@@ -1,27 +1,81 @@
...
+        }
+      );
+      console.log(response.status);
+
+      if (response.status === 201) {
+        notesDispatch({ type: "ARCHIVE_NOTE", payload: noteContent });
+      }
+    } 
+    catch (error) {
+      console.log(error.response);
```
Comment (vaibhav-mahalle) [inline src/component/notecard/Notecard.js:38]: remove console.log() while raising PR

**Review by vaibhav-mahalle — APPROVED** [review]
Things I liked while reviewing the pr:
1. Concise and clear commit messages
2. clarity in code structure

Things you could improve:

1. remove console.log() from the code

File: src/component/notecard/Notecard.js (hunk 1 above)
Comment (shubhamkumar648) [reply src/component/notecard/Notecard.js:38]: this console for error when api response fail

File: src/component/archives/archived.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.main-noteCard-container {
+
+    width: 20rem;
+    padding: 1rem;
```
Comment (rehanmohiuddin) [inline src/component/archives/archived.css:4]: make some standard css classes instead of hard coding them

File: src/component/edit/edit.css [hunk 3]
```diff
@@ -4,12 +4,14 @@
...
-    background-color: #00000099;
+    background-color:  hsla(0, 0%, 0%, 0.6);
     z-index: 1;
 }
 
 .edit-model {
     position: absolute;
     top: 27%;
     left: 36%;
+    background-color: #f0f8ff;
```
Comment (rehanmohiuddin) [inline src/component/edit/edit.css:15]: make color variables

File: src/component/filter/Filter.js [hunk 4]
```diff
@@ -202,7 +202,7 @@ export const Filter = () => {
             </div>
 
             <button
-              className="btn btn__secondary"
+              className="btn btn__primary"
               onClick={() => dispatchFilter({ type: "CLEAR_ALL" })}
```
Comment (rehanmohiuddin) [inline src/component/filter/Filter.js:206]: make constants instead of Hard coding them

File: src/component/notecard/Notecard.js [hunk 5]
```diff
@@ -1,27 +1,81 @@
...
   const [isEdit, setisEdit] = useState(false);
 
+  const { encodedToken } = useAuth();
+  const { notesDispatch } = useNotes();
+
+  const archivedHandler = async () => {
+
+     try 
+    {
+      const response = await axios.post(
```
Comment (rehanmohiuddin) [inline src/component/notecard/Notecard.js:23]: add axios request in seperate services folder just a suggestion

File: src/reducer/Notereducer.js [hunk 6]
```diff
@@ -13,6 +16,43 @@ export const NoteReducer = (state, action) => {
         ),
       };
 
+    case "ARCHIVE_NOTE":
```
Comment (rehanmohiuddin) [inline src/reducer/Notereducer.js:19]: make constants

**Review by rehanmohiuddin — APPROVED** [review]
suggested changes all are good just go through it

**Pull request merged**